CONTRIBUTING.md

```diff
@@ -1,9 +1,23 @@
 # Contribution Guidelines
 
-Follow the [Code of Conduct][1].
+All contributors must follow the [Code of Conduct][1].
 
 [1]: /CODE_OF_CONDUCT.md
 
+## Join the Community
+
+Users of the University of Illinois GitHub Service are encouraged to join the 
+conversations in the [Community Discussions][3].
+
+[3]: https://github.com/uillinois-community/uillinois-community.github.io/discussions
```

Comment:
Would you mind renumbering link references 2-4 so they're in order? Otherwise I think this covers what I was after